helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/ticket/AcceptTicketService.java

```diff
@@ -45,12 +46,16 @@
 
   private final HelpDeskAi helpDeskAi;
 
+  private final HelpDeskBotInfo helpDeskBotInfo;
```

Comment:
I don't think you need this field. You can get this information from 'HelpDeskBotConfig.getHelpDeskBotUrl()' method.